### PR TITLE
Rfm-1

### PR DESCRIPTION
**Pull request by BernieHuang2008**

### CODE REVIEW

File: RequestForMoney/1/request.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# 物品清单
+|物品|单价/元|数量|总价/元|用途实用性证明|
+|---|---|---|---|---|
+|乐青云,香港 CNI 型服务器|19.90/月|3|59.70|见[附录1](appendix/1.md)|
+
+总计：59.70 元
+
+# 审批
+部门负责人：
```
Comment (cxzlw) [inline RequestForMoney/1/request.md:18]:
```suggestion
部门负责人：王思诚
```

**Review by cxzlw — CHANGES_REQUESTED** [review]

**Review by cxzlw — APPROVED** [review]

**Comment by cxzlw** [comment]
lgtm

File: RequestForMoney/1/request.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+审核时间：
+
+---
+经费审批人：
```
Comment (mishmichmeng) [inline RequestForMoney/1/request.md:23]: 经费审批人：孟思言

File: RequestForMoney/1/request.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+总计：59.70 元
+
+# 审批
+部门负责人：王思诚
+
+审核时间：2024/07/24 22:07
+
+---
+经费审批人：
```
Comment (mishmichmeng) [inline RequestForMoney/1/request.md:23]:
```suggestion
经费审批人：孟思言
```

File: RequestForMoney/1/request.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+审批时间：
```
Comment (mishmichmeng) [inline RequestForMoney/1/request.md:25]:
```suggestion
审批时间：2024/07/26 21：50
```

**Comment by BernieHuang2008** [comment]
@mishmichmeng 下次顺便给个approval

**Pull request merged**